lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -23,13 +33,6 @@
         <tbody></tbody>
       </table>
     </div>
-    <div class="bottom-bar">
-      <label>
-          <span class="label-text sr">{{add_placeholder}}</span>
-          <input type="text" id="add-field" name="add-field" class="add-field" placeholder="{{add_placeholder}}">
-      </label>
-      <input type="button" name="add" class="add" value="{{add_btn_label}}">
-    </div>
```

Comment:
@lukyfebrianto sorry for the bad descriptions, but could you leave the button at the bottom of the list and add one more at the top?

Reply:
@polesye  so we have 2 button(top and bottom) right ?

Reply:
yes

Reply:
Agree to have 2 buttons 'cause as i remember edx do not have pagination for emails list

Reply:
done @polesye , @TamoshaytisV 
![screenshot from 2017-11-02 11-10-48](https://user-images.githubusercontent.com/25189966/32308011-81fd5fb4-bfbe-11e7-9620-a26dbe2f5a06.png)